Library/Formula/powerline.rb

```diff
@@ -0,0 +1,23 @@
+class Powerline < Formula
+  desc "statusline plugin for vim, and provides statuslines & prompts"
+  homepage "https://github.com/powerline/powerline"
+  url "https://github.com/powerline/powerline/archive/#{version}.tar.gz"
+  version "2.3"
+  sha256 "c65762df4733afed585af722978d24bc555fbbabb8ae707e4e525dff6412de5f"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    system "python", *Language::Python.setup_install_args(prefix)
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "#{bin}/powerline", "--help"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!